opennms-provision/opennms-provision-persistence/src/main/java/org/opennms/netmgt/provision/persist/requisition/Requisition.java

```diff
@@ -464,9 +466,12 @@ public void validate() throws ValidationException {
     	if (m_foreignSource == null) {
     	    throw new ValidationException("Requisition 'foreign-source' must be set!");
     	}
-    	if (m_foreignSource.contains("/")) {
-            throw new ValidationException("Foreign Source (" + m_foreignSource + ") contains invalid characters. ('/' is forbidden.)");
-    	}
+
+        for(String illegalCharacter : ILLEGAL_CHARACTERS) {
```

Comment:
If there are requisitions already there with `imported:` as foreign source, they will not be processed now, right ?
I think we prevent requisitions with `:` in their name from web ui, but not from xml. 

May be we should add this as a breaking change. wdyt ?

Reply:
Sure.